documentation/Makefile

```diff
@@ -21,7 +21,7 @@ help: .venv
 	python3 -m venv .venv
 	.venv/bin/pip -q install --upgrade pip
 	.venv/bin/pip -q install -r requirements.txt
-	.venv/bin/pip -q install ..
+	.venv/bin/pip -q install -e ..
```

Comment:
Note that this isn't used in CI. Is it needed locally either?

Reply:
I added it for use locally. Otherwise you would have to rebuild the virtualenv every time you rebuilt the documentation locally. Locally its building documentation hy importing the cocotb installation in the virtualenv. That's why I was questioning the CWD of towncrier.